pkg/config/config.go

```diff
@@ -295,10 +301,17 @@ func (c *Config) validate() error {
 		}
 	}
 
+<<<<<<< HEAD
```

Comment:
This looks like a rebase issue.

Reply:
This seems to be outdated in the PR, all tests are passing now.

Reply:
It's still there in the original commit, but was fixed somewhere later in the PR. I generally want each commit to be compilable and encapsulate an entire change. It makes backporting things go more smoothly.